database/migrations/2022_09_07_141854_rename_capabilities.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class RenameCapabilities extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		// flip the locked value
+		DB::table('users')->update(['is_locked' => DB::raw('NOT is_locked')]);
+
+		// rename the column
+		Schema::table('users', function (Blueprint $table) {
+			$table->renameColumn('is_locked', 'may_edit_own_settings');
```

Comment:
I need to check whether renaming a columns breaks foreign constraints for SQLite. I have that feeling that renaming was mimicked by deleting and re-creating the table and forgetting foreign key constraints on that way.